### PR TITLE
Change remove column call to properly respect method signature for Rails 3 vs Rails 4

### PR DESCRIPTION
**Pull request by andrewhubbs**
This is a fix for issue #1946.

No test included. Was unsure how to properly test Rails version specific behavior. Happy to add a test with some guidance.

### CODE REVIEW

**Comment by maclover7** [comment]
Thanks for submitting this patch! :smiley: 

As far as testing this, you can try and stub out ```ActiveRecord::VERSION::MAJOR``` to equal 4 (or whatever value you'd like) by doing the following in an RSpec test: ```stub_const("ActiveRecord::VERSION::MAJOR", 4)```.

File: lib/paperclip/schema.rb [hunk 1]
```diff
@@ -40,7 +40,11 @@ def remove_attachment(table_name, *attachment_names)
         attachment_names.each do |attachment_name|
           COLUMNS.each_pair do |column_name, column_type|
             column_options = options.merge(options[column_name.to_sym] || {})
-            remove_column(table_name, "#{attachment_name}_#{column_name}", column_type, column_options)
+            if ActiveRecord::VERSION::MAJOR >= 4
+              remove_column(table_name, "#{attachment_name}_#{column_name}", column_type, column_options)
```
Comment (tute) [inline lib/paperclip/schema.rb]: Is `column_type, column_options` needed here? Can we just remove it for any version of Rails?

Comment (tute) [inline lib/paperclip/schema.rb]: We don't need that, see 127e8797fb3c6daa2dbcd6763778d58b6645777f and 6ea7c26512434a93b598602fd6e959452a97969b. Will remove.

**Comment by tute** [comment]
Thank you! Merged into master.

**Pull request closed without merging**